addons/handhelds/stringtable.xml

```diff
@@ -3,61 +3,75 @@
     <Package name="backpacks">
         <Key ID="STR_TFAR_handhelds_RF7800STR">
             <Original>RF-7800S-TR</Original>
```

Comment:
Could you please do me a favor and replace `<Original> *** </Original>` with `<English> *** </English>`
If forgot to change this inside this stringtable